runtime/common/RuntimeMLIR.cpp

```diff
@@ -142,7 +142,7 @@ void applyWriteOnlyAttributes(llvm::Module *llvmModule) {
 // overlap.
 // Reference:
 // https://github.com/qir-alliance/qir-spec/blob/main/specification/under_development/profiles/Base_Profile.md?plain=1#L237
-mlir::LogicalResult verifyMeasurementOrdering(llvm::Module *llvmModule) {
+mlir::LogicalResult verifyBaseProfileMeasOrdering(llvm::Module *llvmModule) {
```

Comment:
nit: could just spell out Measurement since this is a function name.

Reply:
Renamed in f1e6886e.